### PR TITLE
Update German locales

### PR DESCRIPTION
**Pull request by jld3103**

### CODE REVIEW

File: web/locales/de.json [hunk 1]
```diff
@@ -1097,10 +1099,12 @@
...
         "RU-EU": {
-            "zoneName": "Russland"
+            "zoneName": "Russland",
+            "countryName": "Russland"
         },
         "RU-AS": {
-            "zoneName": "Russland"
+            "zoneName": "Russland",
+            "countryName": "Russland"
         },
```
Comment (madsnedergaard) [inline web/locales/de.json:1108]: There's been a few updates to RU zones, would you mind updating the translations to reflect this? Here's the names from `en.json` translation :)

```
		  "RU-EU": {
            "countryName": "Russia",
            "zoneName": "Arctic"
          },
          "RU-AS": {
            "countryName": "Russia",
            "zoneName": "East"
          },
          "RU-FE": {
              "countryName": "Russia",
              "zoneName": "Far East"
          },
          ```

**Review by madsnedergaard — COMMENTED** [review]
Nice, thanks for the contribution! 💪 
I have one comment, otherwise it looks good

File: web/locales/de.json (hunk 1 above)
Comment (jld3103) [reply web/locales/de.json:1108]: Yes, I can do that

**Review by madsnedergaard — APPROVED** [review]
Great! 🙌

**Pull request merged**